cmd/localCI/Makefile

```diff
@@ -0,0 +1,24 @@
+DESTDIR :=
+PREFIX := /usr/local
+BINDIR := $(PREFIX)/bin
+SYSCONFDIR := $(PREFIX)/etc
+LIBEXECDIR := $(PREFIX)/libexec
+
+SOURCES := $(shell find . 2>&1 | grep -E '.*\.(c|h|go)$$')
```

Comment:
Just a note - I'm not a fan of such wildcarding in Makefiles - at some point somebody (usually me ;-) will end up copying a file whilst experimenting, and then spend ages trying to figure out why the system now won't build or fails or ..... due to the wildcarding picking up files that are not actually part of the project.